sope-core/NGExtensions/FdExt.subproj/NSString+Encoding.m

```diff
@@ -97,14 +99,20 @@ + (NSStringEncoding)stringEncodingForEncodingNamed:(NSString *)_encoding {
 + (NSStringEncoding)stringEncodingForEncodingNamed:(NSString *)_encoding {
   NSString *s  = [_encoding lowercaseString];
   unsigned len = [s length];
-  
+
   if (s == nil)
     return 0;
 
   switch(len) {
+  case 0:
```

Comment:
you will return 0 on that case, because switch statement will do nothing and there is a `return 0;` at the end of this method